README.md

```diff
@@ -365,20 +392,22 @@ echo "gruntwork-io/terragrunt gruntwork-io/terratest" | git-xargs \
+| `--skip-pull-requests`                | If you don't want any pull requests opened, but would rather have your changes committed directly to your specified branch, pass this flag. Note that it won't work if your Github repo is configured with branch protections on the branch you're trying to commit directly to!                                                                                                                                              | Boolean | No       |
+| `--skip-archived-repos`               | If you want to exclude archived (read-only) repositories from the list of targeted repos, pass this flag.                                                                                                                                                                                                                                                                                                                     | Boolean | No       |
+| `--dry-run`                           | If you are in the process of testing out `git-xargs` or your initial set of targeted repos, but you don't want to make any changes via the Github API (pushing your local changes or opening pull requests) you can pass the dry-run flag. This is useful because the output report will still tell you which repos would have been affected, without actually making changes via the Github API to your remote repositories. | Boolean | No       |
+| `--max-concurrent-repos`              | Limits the number of concurrent processed repositories. This is only useful if you encounter issues and need throttling when running on a very large number of repos. Default is `0` (Unlimited)                                                                                                                                                                                                                              | Integer | No       |
+| `--draft`                             | Whether to open pull requests in draft mode. Draft pull requests are available for public GitHub repositories and private repositories in GitHub tiered accounts. See [Draft Pull Requests](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) for more details.                                                   | Boolean | No       |
+| `--seconds-between-prs`               | The number of seconds to wait between opening serial pull requests. If you are being rate limited, continue to increase this value until rate limiting eases. Note, this value cannot be negative, so if you pass a value less than 1, the seconds to wait between pull requests will be set to 1 second.                                                                                                                     | Integer | No       |
```

Comment:
Could you incorporate `Default: 1` here like for `--loglevel`?

Reply:
NIT: In fact for all the non-required args, it would be nice to say what the default is explicitly.

Reply:
Agreed! I'll add this in.

Reply:
Done in [57526e2](https://github.com/gruntwork-io/git-xargs/pull/67/commits/57526e26aa4aa61740fbbba2476fcbfe1535ea92)